agent/tool-scripts/postprocess/unittests

```diff
@@ -205,8 +216,13 @@ for i in $(ls -1 samples/) ; do
             testhaproxy_ocp $i
             sts=$?
             ;;
+<<<<<<< HEAD
```

Comment:
Looks like you did not resolve this merge conflict.  Would you like some help with this?

Reply:
missed that one.